src/quo2/components/icons/svg.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.icons.svg
+  "Declare icons in this namespace when they have two possible colors, because the
+  ReactNative `:tint-color` prop affects all non-transparent pixels of PNGs. If
+  the icon has only one color, prefer a PNG.
+
+  Keep all SVG components private and expose them by name in the `icons` var."
+  (:require [react-native.svg :as svg]
+            [quo2.foundations.colors :as colors]))
+
+(defn- container
+  [{:keys [size accessibility-label style]
+    :or   {size 20}}
+   & children]
+  (into [svg/svg
+         {:accessibility-label accessibility-label
+          :style               style
+          :width               size
+          :height              size
+          :view-box            (str "0 0 " size " " size)
+          :fill                :none}]
+        children))
+
+(defn- clear-20
+  [{:keys [color color-2] :as props}]
```

Comment:
color 2 is like a background color? 🤔

Reply:
I initially tried to name `color-2` as `background-color`, just to realize it was more confusing for some icons, like `mutual-contact`, `positive-state`, etc, especially when/if we migrate more icons to SVGs I think `color-2` is more future proof. It's really like a secondary color, but I didn't want to name it `secondary-color` or `alternate-color`, etc, so I ended up going for this simplistic name.